src/abstractdataframe/show.jl

```diff
@@ -556,32 +574,77 @@ end
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
 #' showall(df, true)
 function Base.showall(df::AbstractDataFrame,
-                      splitchunks::Bool = false) # -> Void
-    showall(STDOUT, df, splitchunks)
+                      splitchunks::Bool = false,
+                      onechunk::Bool = false) # -> Void
+    showall(STDOUT, df, splitchunks, onechunk)
     return
 end
 
+#' @exported
 #' @description
 #'
 #' Render a summary of the column names, column types and column missingness
 #' count.
 #'
 #' @param io::IO The `io` to be rendered to.
 #' @param df::AbstractDataFrame An AbstractDataFrame.
+#' @param allcols::Bool If `false` (default) only a subset of columns
+#'        fitting on the screen is printed
+#' @param values::Bool If `true` (default) first and last value of
+#'        each column is printed
 #'
 #' @returns o::Void A `nothing` value.
 #'
 #' @examples
 #'
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
-#' showcols(df, true)
-function showcols(io::IO, df::AbstractDataFrame) # -> Void
+#' showcols(STDOUT, df)
```

Comment:
I thould it should stay as the example with implicit `STDOUT` is given below for definition of `function showcols(df::AbstractDataFrame, allcols::Bool=false, values::Bool=true)`.